src/selvage/test/flow.clj

```diff
@@ -0,0 +1,199 @@
+(ns selvage.test.flow
```

Comment:
when deciding names for namespaces that implement `clojure.test` functionality, I can never think of a good name. Like `selvage.clojure.test.flow` isn't good, and nor is `selvage.test.flow` because it makes me think this is a test namespace.
Any ideas?